src/lib/stack.js

```diff
@@ -61,7 +61,13 @@ function getDiskQuota () {
 
     return res.json()
   })
-  .then(json => parseInt(json.data.attributes.quota, 10))
+  .then(json => {
+    if (Number.isInteger(json.data.attributes.quota)) {
+      return parseInt(json.data.attributes.quota, 10)
+    } else {
+      return 100000000000 // @TODO Waiting for instructions about how to deal with limitless instances
```

Comment:
To clarify that value, could you just use instead declared at the top of the file, with an explicit name, for example
```js
const DEFAULT_DISK_QUOTA = 100000000000
```